### DISCUSSION TITLE
ASreview results

### DISCUSSION THREAD

**Discussion by haneen81** [asreview/asreview#1899] [Q&A]
I have used ASreview to validate scoping review results, I have used Oracle mode, but the results 100%  match with the manual study that I did by using Excel, Do you have any comments please, is there anything wrong in the process, considering that I have used title and abstract to make validation ?

**Comment by RiannevanderFeen** [comment]
@haneen81, I am curious about what your total dataset amounts to.... As it sounds like you might have used many parameters (such as peer reviewed, publication date, academic journal) in the manual search (as you might have been used to) and then used the same dataset in ASReview. Is that correct? 

What we did (for our scoping review) was use the manual search for prior knowledge puposes, and then not use too many parameters in the database and use as many synonyms as possible in the search strings. Because, we learned from the ASReview Lab summer course that the existing databases are not as precise as the active learning models in ASReview Lab. Also, type-o's etc. can cause a false exclusion in your search in the database. If this makes sense...

**Comment by lastoel** [comment]
Hi @haneen81 - thanks for your question. I hope I understand it correctly. 

I essentially have the same follow-up question as Rianne: did you enter the same dataset that you reviewed manually into ASReview? 

If that is the case, you can expect ASReview to give the same results as your manual study, as the programme is simply presenting you with the same information, but in a different order; namely with the more relevant records appearing first. 

The power of ASReview lies in its ability to help you find the most likely relevant records out of a _large amount of records_. If you want to use ASReview to see if your scoping review covers the full range of the topic you're researching, you could use a broader set of search strings in your databases and enter the resulting, large set of records into ASReview. The AI will likely present you with most of the same records you have also labeled 'relevant' in your manual study, but it might also present you with other, potentially relevant records you hadn't included in your initial screening! The fewer 'new' relevant records you find this way, the stronger the evidence that your initial scoping review was complete. 

I hope I've understood your question correctly and that this answer is of some use!
